org.eclipse.sprotty/src/main/java/org/eclipse/sprotty/ILayoutEngine.java

```diff
@@ -17,7 +17,7 @@
 
 /**
  * A layout engine is able to compute layout information for a model. Invoked by {@link DefaultDiagramServer}
- * when {@link DefaultDiagramServer#needsServerLayout(SModelRoot)} returns {@code true}.
+ * when {@link DefaultDiagramServer#getServerLayoutKind(SModelRoot)} returns {@code true}.
```

Comment:
The method no longer returns `boolean`, but an enum.

Reply:
Fixed